packages/vm/test/t8n/helpers.ts

```diff
@@ -124,3 +133,71 @@ export function normalizeNumbers(input: any) {
   }
   return input
 }
+
+/**
+ * Formats an individual EVM step trace as a JSON object
+ * @param step an {@link InterpreterStep} emitted by the EVM `step` event
+ * @param memory whether to include the memory in the trace
+ * @returns a JSON object that matches the EIP-7756 trace format
+ */
+export const stepTraceJSON = (
```

Comment:
Can't we directly JSON.stringify the output of the step event?

Reply:
Yes, was on the fence about whether to do that or not since it would return a string instead of a JSON object.

Reply:
Ah right, yes sorry my point was that we need this helper method here. Should this logic not directly be integrated into the `step` event? Such that if we want to consume it, we can directly use the output of the step event. Such that if you are listening, you can write it to a file by `JSON.stringify` the object output of the step event, or use the object directly (so no additional helper stuff needed as written here)

Reply:
Ah, I misunderstood your question. The `InterpreterStep` interface has `bigint` and `Address` typed elements in it that aren't directly convertible to JSON so we have to do some handling regardless.  And since the event has additional elements in it beyond just the basic trace definition, we don't want to dump all of this directly into a trace.